modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -1064,17 +1138,22 @@ class NDB_Form_Statistics extends NDB_Form
         $this->tpl_data['Sites']    = $sites;
 
         $this->tpl_data['reliability_completion']
-            = $this->getReliabilityData($Param_Project, $Param_Site);
+            = $this->getReliabilityData(
+                $Param_Project,
+                $suproject_query,
+                $Param_Site
+            );
     }
     /**
      * GetReliabilityData function
      *
-     * @param string $Param_Project the value of Param_Project
-     * @param string $Param_Site    the value of Param_Site
+     * @param string $Param_Project   the value of Param_Project
+     * @param string $suproject_query the value of suproject_query
```

Comment:
I think `suproject` is a typo, but as long as you're changing these can you also clean up the description comment for the other two? (Of course any variable has the value of the variable! But what's the meaning of the variable supposed to represent?)